### DISCUSSION TITLE
petition

### DISCUSSION THREAD

**Discussion by Windows1Door2Daniel** [FunkinCrew/Funkin#1632] [Ideas]
petition to eliminate the negative points

is very anoying that you complete a week but with negative points

the week record will be 0 points

![Captura de pantalla (134)](https://user-images.githubusercontent.com/81389527/114130812-33797600-98b6-11eb-8313-f0611f491c20.png)

**Comment by helpmeiguess** [comment]
excuse me. how do you accidently beat a week with negative points?

**Comment by isakube** [comment]
ballistic?

**Comment by Windows1Door2Daniel** [comment]
blammed, milf hard

also ballistic

**Comment by aflacc** [comment]
You'd need to miss so many times for every note you hit to keep alive. I don't see how you'd be able to beat any level with negative points
